service/controller/resource/tccpn/template/template_main_record_sets.go

```diff
@@ -0,0 +1,19 @@
+package template
+
+const TemplateMainRecordSets = `
+{{- define "record_sets" -}}
+{{- $v := .RecordSets -}}
+{{- if $v.Route53Enabled -}}
+{{ range $r := $v.Records }}
+  {{ $r.Resource }}:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      ResourceRecords:
+      - !GetAtt {{ $r.ENI.Resource }}.PrimaryPrivateIpAddress
+      Name: '{{ $r.Value }}.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
+      HostedZoneId: {{ $v.HostedZoneID }}
+      Type: A
```

Comment:
This does not work right now and the TCCPN stack ends up in `CREATE_FAILED`. I am not sure if we even need that for single master setups, because we did not have it so far. I found [some AWS Docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-recordset.html) but am not sure if the definition really lacks `TTL`. 

> Invalid request: Expected exactly one of [AliasTarget, all of [TTL, and ResourceRecords], or TrafficPolicyInstanceId], but found none in Change with [Action=CREATE, Name=etcd.8ttj5.k8s.ginger.eu-west-1.aws.gigantic.io., Type=A, SetIdentifier=null] (Service: AmazonRoute53; Status Code: 400; Error Code: InvalidInput; Request ID: b705f58d-536f-4f49-9d50-dcca1cbd72e2)

/cc @calvix

Reply:
I will check, we might not need it for a single master but anyway the error will be there for 3  masters so sorting now will help us :)

Reply:
Makes sense. If you like you can take the PR for testing and we can then talk about merging into the feature branch or what else has to be done.